tests/runtime.rs

```diff
@@ -0,0 +1,17 @@
+#[cfg(feature = "e2e-tests")]
+#[test]
```

Comment:
does this annotation here means that all tests will report as a "single test" from "Cargo test" perspective? could we somehow force it to report each test as its own?

Reply:
thanks Ryan.

Reply:
or https://docs.rs/libtest-mimic/ which the ui-testing crate uses.